### PR TITLE
EASY-2451: fix SOLR schema and introduce delete command

### PR DESCRIPTION
**Pull request by rvanheest**
Fixes EASY-2451

#### When applied it will
* fix the SOLR schema after #33 (03bb358)
* refactored the command line interface (095c419)
* introduce a `delete` command for the command line to be able to clear the cache (916e0eb)
* make sure `commit` is always called after `submit` (742dd50)
* fix errors in the logback configuration (d85f702)
* some minor formatting (de77fb9, 34516f3) and rewriting (8a8522c, e277522, 94d5cc2)

@DANS-KNAW/easy for review

#### Notes for deploying
* build and deploy `easy-auth-info` as normal
* ssh into the server as `root` user
* run the following commands (_taken from the LDAP role `easy-solr-search2`, which is not able to be run because there is no version upgrade of SOLR_)
    * `cp /opt/dans.knaw.nl/easy-auth-info/install/authinfo/schema.xml /var/opt/org.apache/solr-home/data/authinfo/conf/`
    * `systemctl restart solr`

### CODE REVIEW

File: src/main/scala/nl.knaw.dans.easy.authinfo/EasyAuthInfoApp.scala [hunk 1]
```diff
@@ -86,7 +104,7 @@ trait EasyAuthInfoApp extends AutoCloseable with DebugEnhancedLogging with Appli
     for {
       ddm <- bagStore.loadDDM(bagId)
       ddmProfile <- getTag(ddm, "profile", bagId)
-      dcmiMetadata = getOptionalTag(ddm, "dcmiMetadata")
+      dcmiMetadata = (ddm \ "dcmiMetadata").headOption
```
Comment (jo-pol) [inline src/main/scala/nl.knaw.dans.easy.authinfo/EasyAuthInfoApp.scala]: I suppose this method used to be extracted to mirror `getTag`.

Comment (rvanheest) [reply src/main/scala/nl.knaw.dans.easy.authinfo/EasyAuthInfoApp.scala]: yes, but it was only used once, so it makes more sense to just have it inline for now.

File: src/main/scala/nl.knaw.dans.easy.authinfo/CommandLineOptions.scala [hunk 2]
```diff
@@ -28,8 +28,9 @@ class CommandLineOptions(args: Array[String], configuration: Configuration) exte
   val description: String = s"""Provides consolidated authorization info about items in a bag store."""
   val synopsis: String =
     s"""
-       |  $printedName run-service # Runs the program as a service
-       |  $printedName <item-id> # Retrieves the information from the cache or directly from the bag-store""".stripMargin
+       |  $printedName get <item-id> # Retrieves the information from the cache or directly from the bag-store
+       |  $printedName delete <solr-query> # Delete the selected item(s) from the SOLR index
```
Comment (vesaakerman) [inline src/main/scala/nl.knaw.dans.easy.authinfo/CommandLineOptions.scala]: It would be nice to have an example of a solr-query here

Comment (rvanheest) [reply src/main/scala/nl.knaw.dans.easy.authinfo/CommandLineOptions.scala]: I know, but I was hoping no one would notice 😉
I'm not so proficient with SOLR. The oly query I know is `"*"` to delete everything. But I'll see what I can do!

**Pull request merged**